content/en/user-guide/tools/cockpit/index.md

```diff
@@ -26,63 +23,46 @@ On [Windows](https://www.windowscentral.com/how-disable-smartscreen-trusted-app-
 
-### Automatic environment check
+### Automated environment checks
 
-When the Cockpit starts it will automatically check your system environment whether everything is ready to start LocalStack.
-It will also download the LocalStack Docker image for you, should it not be on your system.
+While starting Cockpit, it will automatically check your system environment to ensure that everything is ready to start LocalStack. It includes checking Docker version, pulling the official LocalStack Docker image, and checking for the presence of the LocalStack CLI and runtime status. In the absence of a Docker image, Cockpit will automatically pull the latest version of the LocalStack Docker image.
 
 <p>
 {{< img src="cockpit-init-check.png" class="img-fluid shadow rounded" >}}
 </p>
 
 ### Run configurations
 
-Manage and select LocalStack run configurations to start LocalStack with a particular configuration.
-Save your LocalStack Pro API key, or a particular set of environment variables into a run configuration. 
-
-<p>
-{{< img src="cockpit-runconfig-select.png" class="img-fluid shadow rounded" >}}
-</p>
+Using Cockpit, you can manage and select LocalStack run configurations to start LocalStack with a particular configuration. Configurations allow you to save your LocalStack Pro API key, or a particular set of environment variables into a run configuration.
 
 <p>
 {{< img src="cockpit-runconfig-edit.png" class="img-fluid shadow rounded" >}}
 </p>
 
 ### Manage your LocalStack instance
 
-Start and stop LocalStack by simply clicking a button.
-No mucking about in the CLI or a docker-compose file.
-The environment and Services screen give you instant insights into your running instance.
-
-<p>
-{{< img src="cockpit-localstack-start.gif" class="img-fluid shadow rounded" >}}
-</p>
+You can easily start and stop your LocalStack instance with a single click. Cockpit also provides you with the status of your LocalStack instance, including the number of running services and the runtime information.
 
 ### Quick log access
 
-Get quick access to your LocalStack logs for instant insights.
-
-*Note*: The beta version does not yet have auto-followcockpit, so you need to click "Refresh" and "Scroll to end".
+Cockpit provides quick access to your LocalStack logs for instant insights. Currently, the beta version of Cockpit does not have auto-follow cockpit, so you need to click "Refresh" and "Scroll to end" to view the latest logs.
 
 <p>
 {{< img src="cockpit-logs.png" class="img-fluid shadow rounded" >}}
 </p>
 
 ## Known issues
 
-- MacOS ≤ v10 not supported yet.
-- There may be glibc issues on older Linux versions.
+- Mac OS X 10.0 and lower are not supported yet.
+- There may be `glibc` issues on older Linux versions.
 
-## Report issues
+While installing Cockpit on Mac M1, you may encounter an `xattr` error. To resolve this issue, you can visit the [Apple Developer Forum thread](https://developer.apple.com/forums/thread/692774).
```

Comment:
technically it's not an "xattr error", mac os may prevent the software from running because of issues with code signing. to fix it, you can run `xattr -rc` in the terminal on the binary file and then run the application again.